### PR TITLE
[Items] CAES-1538: Fix domains for favicons

### PR DESCRIPTION
**Pull request by sa715**
Signed-off-by: Sergey Alekseyev <sa@4xxi.com>

### CODE REVIEW

File: packages/components/List/Item/index.js [hunk 1]
```diff
@@ -21,13 +22,16 @@ import {
 } from './styles';
 
 const ItemIcon = ({ website, type }) => {
+  console.log(website);
```
Comment (cu4xxi) [inline packages/components/List/Item/index.js:25]: Redundant console

**Review by cu4xxi — APPROVED** [review]

File: packages/components/List/Item/index.js (hunk 1 above)
Comment (sa715) [reply packages/components/List/Item/index.js:25]: Removed

File: packages/common/utils/url.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  if (!/^https:\/\//.test(url)) {
+    prefix = 'https://';
+  } 
+    
+  try {
+    const urlObject = new URL(prefix + url);
+    
+    return urlObject;
+  } catch(error) {
+    console.warn(`URL ${url} is invalid`);
```
Comment (beshkenadze) [inline packages/common/utils/url.js:13]: You should avoid such warns because of spamming to logs when we have a lot of wrong items.

**Review by beshkenadze — APPROVED** [review]

File: packages/common/utils/url.js (hunk 2 above)
Comment (sa715) [reply packages/common/utils/url.js:13]: Removed

**Pull request merged**